fighthealthinsurance/generate_prior_auth.py

```
@@ -0,0 +1,238 @@
+                return {
+                    "error": f"Failed to generate proposal text with model {index+1}"
+                }
+
+            # Create a unique ID for this proposal
+            proposed_id = uuid.uuid4()
+
+            # Create and save the proposal in the database, with sqlite this can result in db locked errors.
+            try:
+                await self._create_proposal(prior_auth, proposed_id, proposal_text)
+            except:
+                pass
```

Ruff (0.8.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)